integration-tests/features/steps/stack_analysis.py

```diff
@@ -800,3 +809,50 @@ def validate_topic_list(context, key):
         for dep in deps:
             assert len(dep['topic_list']) == len(input_stack_topics[dep['name']])
             assert sorted(dep['topic_list']) == sorted(input_stack_topics[dep['name']])
+
+
+@when('I tried to fetch dynamic manifests from s3')
+def dynamic_manifest_file(context):
+    """Integration for dynamically generated manifests files."""
+    response, status = context.s3interface.get_object_from_s3(bucket_name=context.reports_bucket)
+    assert status == 200, response
+
+
+@then('I should be able to validate them')
+def validate_dynamic_manifest_file(context):
```

Comment:
It is not validation step, right? It just retrieves data from S3 w/o any validation

Reply:
@tisnik .. List of Manifests are received from S3, So, this function randomly selects one stack of them and generate a valid manifest out of it.